src/lib/utils.js

```diff
@@ -1,25 +1,3 @@
-/**
```

Comment:
question: (this is a comment for the whole file) Why is this file chaned this much?

A formatter maybe?

Reply:
I decided to order everything alphabetically without asking anyone :smile:

I'll rollback the ordering!